consensus/src/chained_bft/chained_bft_smr_test.rs

```diff
@@ -124,6 +124,7 @@ impl SMRNode {
         let ValidatorSwarm {
             mut nodes,
             validator_set,
+            ..
```

Comment:
hmm....
".." versus "_"
should we unify? I haven't seen ".." before.

Reply:
When unpacking a struct you can use `..` to ignore all other fields. If you wanted to use `_`, you would have to name the other fields:

```
let ValidatorSwarm {
    mut nodes,
    validator_set,
    discovery_set: _,
} = /* .. */
```

this not too much of a burden for this specific struct, but it's definitely helpful for larger structs.